### PR TITLE
remove https://github.com/hznupeter/blynk-library-for-chinese

### PR DESCRIPTION
**Pull request by hznupeter**

### CODE REVIEW

**Review by per1234 — COMMENTED** [review]
Hello @hznupeter. I am confirming your request.

Unlike submissions, this type of request is still handled manually. The Library Manager index maintainer occasionally comes through to clear all the outstanding requests. They will merge this pull request when it is done. You will get a notification from GitHub at that time.

**Review by MatteoPologruto — APPROVED** [review]
Done:
```
2024/04/04 14:47:38 Loaded 6807 libraries from DB
Removing Blynk For Chinese
warning: Library Git clone folder /home/ubuntu/workspace/gitclones/github.com/hznupeter/blynk-library-for-chinese not present
Success!
```

**Pull request merged**
